client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import "./Header.scss";
+
+interface IProps {
+    userInfo?: { //temporary put ? to use mocks inside component
+        name: string,
+        image: string
+    },
+    movies?: Array<string>,
+    tv?: Array<string>,
+    ratings?: Array<string>
+};
+
+const ArrowDown = () => {
+    return (
+        <svg className="arrowDown " width="8" height="5" viewBox="0 0 8 5" fillOpacity="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Icon's code shouldn't be included in component file. Save it as .svg image in assets directory